### PR TITLE
Add no-response workflow

### PR DESCRIPTION
**Pull request by mhmdanas**
#### What is it?
- [x] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

Add a workflow to help manage issues where the author hasn't responded to requests for more information. Here's an example:
1. I open an issue, but it doesn't give enough information
2. `@opusforlife` creates a comment asking me for the missing enough and labels the issue as `waiting-for-author`
3. If I reply before 14 days, the label is automatically removed. If I reply after 14 days, the label is automatically removed and the issue is re-opened.

I [opened an issue on the action's repo asking whether this works with PRs](https://github.com/lee-dohm/no-response/issues/171), but there's no reply yet. In any case, usage with PRs right now should be avoided, as 14 days is a far too short span of time. I plan to modify the action so we can have a separate `daysUntilClose` for PRs, but for now, this should be used with issues only.

I [created a repo with an identical workflow to the one added in this PR](https://github.com/mhmdanas/test-no-response-action). If anyone is interested in testing, feel free to ask and I'll send you an invite.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: .github/workflows/no-response.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: No Response
+
+# Both `issue_comment` and `scheduled` event types are required for this Action
+# to work properly.
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Schedule for five minutes after the hour, every hour
+    - cron: '5 * * * *'
```
Comment (mhmdanas) [inline .github/workflows/no-response.yml:10]: Maybe make it run every 12 hours (or even only once a day)? An interval of one hour between each run sounds unnecessary and wasteful to me.

**Comment by Stypox** [comment]
I'd add `closeComment: false` so that the issue is closed without any comment that would ping anyone involved in the thread.

**Comment by mhmdanas** [comment]
> that would ping anyone involved in the thread.

It doesn't though?

If you're referring to notifications, you get a notification even if the issue is only closed, so it doesn't matter much. If you're referring to actual pinging (á la `@`), then the comment doesn't contain any pings by default. I think the comment is useful for clarification on why the issue was closed, but I wouldn't entirely oppose removing it if that's the consensus.

**Comment by Stypox** [comment]
> you get a notification even if the issue is only closed

Oh, then nevermind

File: .github/workflows/no-response.yml (hunk 1 above)
Comment (TobiGr) [reply .github/workflows/no-response.yml:10]: once a day is good enough. We just want to remind a few people to answer

**Comment by opusforlife2** [comment]
The run frequency should depend on how frequently the issue maintainers (mostly you) triage issues. If you do it every day, then the workflow should run every day or once in 2 days. This way, the max gap between you applying the label and the issue being closed would be 14-15 days. That is easily acceptable.

Compare this to a frequency of 5 days. The issue might get closed 19-20 days later.

**Comment by mhmdanas** [comment]
Regardless of my triaging frequency, I think once per day is good enough.

**Review by Stypox — APPROVED** [review]

**Review by TobiGr — APPROVED** [review]
Looks good to me now.
Let's give it a try!

**Pull request merged**